packages/saslprep/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@mongodb-js/saslprep",
+  "description": "SASLprep: Stringprep Profile for User Names and Passwords, rfc4013",
+  "keywords": [
+    "sasl",
+    "saslprep",
+    "stringprep",
+    "rfc4013",
+    "4013"
+  ],
+  "author": "Dmitry Tsvettsikh <me@reklatsmasters.com>",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared/tree/main/packages/saslprep",
+  "version": "1.0.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-shared.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "license": "MIT",
+  "exports": {
+    "require": "./dist/index.js",
+    "import": "./dist/.esm-wrapper.mjs"
+  },
```

Comment:
If you use ESM, then you don't get type definitions. I think we need to make this similar to BSON: https://github.com/mongodb/js-bson/blob/main/package.json#L80-L91

Reply:
Fwiw, this is something that looks the same in every package.json in our repos here, we haven’t been seeing any issues ourselves :slightly_smiling_face:

Reply:
(But still happy to address this, of course, done in c261b1e9e7c6ddfd6b2f838ff2b54ad61039ee74)

Reply:
I tried it locally with the module: node16 setting, I think it's specifically that setting that causes the issue 🤷🏻 ty!